estate/models/estate_property.py

```diff
@@ -0,0 +1,39 @@
+from dateutil.relativedelta import relativedelta
+from odoo import fields, models
+
+
+class TestModel(models.Model):
+    _name = 'estate.property'
+    _description = 'It allows to manage your properties'
+    name = fields.Char(required=True, default='Unknown')
```

Comment:
```suggestion
    _description = 'It allows to manage your properties'
    name = fields.Char(required=True, default='Unknown')
```
There should be an empty space here to distinguish private attributes from fields